schema/property.go

```diff
@@ -58,7 +65,7 @@ func NewProperty(id, title, description, typeID, format, relation, relationColum
 
 //NewPropertyFromObj make Property  from obj
 func NewPropertyFromObj(id string, rawTypeData interface{}, required bool) *Property {
-	typeData := rawTypeData.(map[string]interface{})
+	typeData, _ := rawTypeData.(map[string]interface{})
```

Comment:
nit: there's no point in consuming this boolean value and then ignore it. If the reason for this change was consistency please consider removing it from the rest rather than adding it here.

Reply:
Reverted to how it was before